### PR TITLE
Cli import

### PR DESCRIPTION
**Pull request by sbesson**
Following https://github.com/openmicroscopy/ome-documentation/pull/1151, this PR reviews the full page about the command-line import.

Summary of changes:
- use the `program/option` markup extensively like other commands /cc @ximenesuk 
- list all help flags including `--java-help`/`--advanced-help`
- add `--skip` argument with a rationale for using them
- move the Feedback arguments to the top-level section
- simplify the Java command line importer section and tries to make it clear it is used by the CLI import

--no-rebase

### CODE REVIEW

**Comment by sbesson** [comment]
@hflynn: with regard to https://github.com/openmicroscopy/ome-documentation/pull/1150, this PR should make it possible to link directly to the new option via ``` :option:`omero import --skip` ```

File: omero/users/cli/import.txt [hunk 1]
```diff
@@ -15,91 +15,136 @@ Import command
...
+        $ bin/omero import broken_image --report --logs --email my.email@domain.com
+
+
+Command Line Importer
+^^^^^^^^^^^^^^^^^^^^^
+
+The |CLI| import plugin calls the
+``ome.formats.importer.cli.CommandLineImporter`` Java class. The Linux
+OMERO.importer also includes and :file:`importer-cli` shell script allowing to
+call the importer directly from Java.Using the importer using the shell script
```
Comment (mtbc) [inline omero/users/cli/import.txt]: space after period

File: omero/users/cli/import.txt [hunk 2]
```diff
@@ -15,91 +15,136 @@ Import command
...
-::
+    Specify optional step to skip during import.
 
-    ./importer-cli -f /path/to/file
+    The import of very large datasets like High-Content Screening data or
+    SPIM data can be time and resource consuming both at the client and at the
+    server level. This option allows the disabling of some non-critical steps
+    and thus faster import of such datasets. The caveat associated with its
+    usage is that some elements are no longer generated at import time and may
+    either be re-generated during client access (e.g. thumbnails) or may need
```
Comment (ximenesuk) [inline omero/users/cli/import.txt]: My concern here is what "manual intervention" means to an ordinary user. What are they supposed to do? Given that after importing a folder with `--skip minmax` I end up with Nine Shades of Grey.

gray

File: omero/users/cli/import.txt [hunk 3]
```diff
@@ -15,91 +15,136 @@ Import command
 .. program:: omero import
 
 :omerocmd:`import` is probably the first command many users will want to use.
```
Comment (ximenesuk) [inline omero/users/cli/import.txt]: Given that this is now on a page of its own this first sentence seems a little out of place. I'd probably just drop the entire sentence.

File: omero/users/cli/import.txt [hunk 4]
```diff
@@ -15,91 +15,136 @@ Import command
...
+    manual intervention to be generated (e.g. minmax).
+    Available options that can be skipped are currently:
 
-or
+    all
+        Skip all optional steps described below
+    checksum
+        Skip checksum calculation on files after transfer
+    minmax
+        Skip computation of the minima and maxima pixel values
```
Comment (ximenesuk) [inline omero/users/cli/import.txt]: Should it be mentioned here of in the paragraph above that this option includes skipping the checksum...

File: omero/users/cli/import.txt [hunk 5]
```diff
@@ -15,91 +15,136 @@ Import command
...
+        $ bin/omero import large_image --skip minmax
 
-will output a list of all the files which would be imported in groups
-separated by "#" comments. Note that this usage does not require a running
-server to be available.
+    Multiple import steps can be skipped by supplying multiple arguments::
 
-Command Line options
-^^^^^^^^^^^^^^^^^^^^
+        $ bin/omero import large_image --skip checksum --skip minmax
```
Comment (ximenesuk) [inline omero/users/cli/import.txt]: ...and thus this is a redundant example?

File: omero/users/cli/import.txt (hunk 4 above)
Comment (sbesson) [inline omero/users/cli/import.txt]: Yes this now skips the pixel checksum. I will clarify

**Comment by sbesson** [comment]
Added the effect of min/max calculation skip to the known limitations section and linked to this limitation in the `--skip minmax` description. /cc @jburel @dominikl @pwalczysko

File: omero/users/cli/import.txt [hunk 6]
```diff
@@ -14,92 +14,147 @@ Import command
...
-::
+    Specify optional step to skip during import.
 
-    ./importer-cli -f /path/to/file
+    The import of very large datasets like High-Content Screening data or
+    SPIM data can be time and resource consuming both at the client and at the
+    server level. This option allows the disabling of some non-critical steps
+    and thus faster import of such datasets. The caveat associated with its
+    usage is that some elements are no longer generated at import time.
+    Some of them will be re-generated during client access e.g. thumbnails.
```
Comment (mtbc) [inline omero/users/cli/import.txt]: still "re-" here of course

**Comment by hflynn** [comment]
is this good to merge now @mtbc ?

**Comment by mtbc** [comment]
I think commas around the parenthetical clause "like thumbnails" would help: then it's clearer that thumbnails are an example of one of the elements, rather than being some kind of simile that evokes the kind of generation we ought to be thinking of.

**Comment by mtbc** [comment]
I've not read the whole thing carefully, but I'm not aware of any blockers to merging.

**Comment by hflynn** [comment]
All fixed now and green on staging. Good to merge once Travis is finished.

**Pull request merged**